include/staff/ticket-view.inc.php

```diff
@@ -397,7 +397,7 @@ function (user) {
                         if ($user) { ?>
                             <a href="tickets.php?<?php echo Http::build_query(array(
                                 'status'=>'open', 'a'=>'search', 'uid'=> $user->getId()
-                            )); ?>" title="<?php echo __('Related Tickets'); ?>"
+                            )); ?>" title="<?php echo __('User Related Tickets'); ?>"
```

Comment:
This can be broken into two parts: `User` and `Related Ticket` using `sprintf()`